ivy-rich.el

```diff
@@ -360,9 +360,25 @@ or /a/…/f.el."
           ;; e.g. magit or dired, set the list-buffers-directory variable
           (buffer-local-value 'list-buffers-directory buffer))))
 
-(defun ivy-rich-switch-buffer-root (candidate)
-  (when-let* ((dir (ivy-rich--switch-buffer-directory candidate)))
-    (unless (or (and (file-remote-p dir)
+(defvar ivy-rich-project-root-cache
```

Comment:
I think `ivy-rich-project-root-cache` should be private , i.e. `ivy-rich--project-root-cache`?

Reply:
Sounds like a good idea; I changed that.